### ISSUE TITLE
Description field of "Export Data" needs an extra row

### ISSUE THREAD

**Issue by Espo02** [NewEconomyMovement/NEMiOSApp#210]
Please add an extra row to the description field of the Export Data.
![photo_2016-05-24_21-47-13](https://cloud.githubusercontent.com/assets/17567386/15517673/53fee3ee-21f9-11e6-9793-58927da6ffe2.jpg)

**Comment by jabo38** [comment]
@Espo02 I think we missed the window for this now sadly.  We will have to think of a different way of saying the same thing to make it fit.  Do you have any suggestions?

**Comment by jabo38** [comment]
Didn't mean that reference, please ignore.

**Comment by fyahfox** [comment]
@jabo38 I disagree here. Finding shorter translations is a lot of work, and we can't guarantee that we find good solutions for every language.

Adding one more line to the headline should be a 30 seconds problem though.

**Comment by fyahfox** [comment]
Ok, I talked to @jabo38 again and what he means is that the Android translation for this headline is way shorter. I don't understand why the devs use totally different wording for the exact same thing. Are you aware that this causes our translators a lot of work?

Anyway, I guess for this situation I could just copy all the translations from Android over to iOS.

**Comment by fyahfox** [comment]
Fix will be visible in one of the next releases (after I exported all the translation tables and sent it to the devs).

**Comment by jabo38** [comment]
latest strings should have been sent but problem still exists

**Comment by fyahfox** [comment]
@jabo38 "note that loc files are not updated yet, we will finish with them tomorrow." ;)

**Comment by jabo38** [comment]
okay. thanks

**Comment by fyahfox** [comment]
is this fixed now?
